joss_paper/paper.md

```diff
@@ -0,0 +1,289 @@
+
+The `sunpy.map` and `sunpy.timeseries` subpackages provide core data types (`Map` and `TimeSeries`, respectively) that are designed to provide a general, standard, and consistent interface for loading and representing solar data across different instruments and missions. These classes load data which conform to solar physics standards and conventions such as FITS [@wells_fits_1981], FITS World Coordinate Systems (WCS) [@fits_wcs], and solar-specific FITS headers [@thompson_coordinates_2006], while allowing customization to account for differences in specific instruments. Visualization methods are also provided to inspect and plot those data. Example visulizations of both `TimeSeries` and `Map` are shown in Figure 1.
+
+![Left: An example of `TimeSeries` for the GOES X-ray Sensor in two broadband channels. Right: A `Map` of the extreme ultraviolet 171 $\mathring A$ channel of AIA corresponding to the time of a solar flare depicted by the vertical dashed line in the left-hand panel.](https://codimd.s3.shivering-isles.com/demo/uploads/upload_5eabd346e54c29b1d9b74aa11e351b30.png)
+
+The `sunpy.coordinates` subpackage provides support for representing and transforming coordinates used in solar physics and astrophysics. These coordinates may represent events (e.g., flares), features on or above the Sun (e.g., magnetic loops), or the position of structures traveling throughout the heliosphere (e.g., coronal mass ejections). The package currently implements the most widely used Sun-centered coordinate frames, and extends `astropy.coordinates`.
+
+Other functionality provided by `sunpy` includes physical models of solar behavior, such as differential rotation, color maps for certain data sources, image-processing routines integrated with `Map`, and useful physical parameters such as constants.
+
+The `sunpy` package is designed to be extensible, which means that it is easy to add support for additional instruments or data sources. It relies heavily on the `astropy` Python package as well as the scientific python stack (e.g. `numpy`, `scipy`, `matplotlib` and `pandas`).
+
+A more complete description of the SunPy Project and the `sunpy` package, the methodology, development model, and implementation can be found in [@sunpy_community2019].
+
+The SunPy Project supports affiliated packages, which build upon or extends the functionality of `sunpy`. Current affiliated packages are `drms` [@Glogowski2019drms], [`ndcube`](https://docs.sunpy.org/projects/ndcube), [`radiospectra`](https://docs.sunpy.org/projects/radiospectra) and [`IRISPy`](https://docs.sunpy.org/projects/irispy). The Project is also a member of the Python in Heliophysics community [PyHC, @annex], whose mission is to enable interdisciplinary analysis across all sub-disciplines of heliophysics by adhering to standards for code development and interoperability.
```

Comment:
You should add a link to `drms` like you have with the other affiliated packages.

Reply:
👍 Fixed by https://github.com/sunpy/sunpy/commit/6188dad4528161f7096954cd569f6d9399d8b697